timelock-impl/src/main/java/com/palantir/atlasdb/timelock/AsyncTimelockService.java

```diff
@@ -50,4 +53,11 @@
     StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransaction(
             StartIdentifiedAtlasDbTransactionRequest request);
 
+    LeasableRefreshLockResponse leasableRefreshLockLeases(Set<LockToken> tokens);
+
+    LeasableStartIdentifiedAtlasDbTransactionResponse leasableStartIdentifiedAtlasDbTransaction(
```

Comment:
Within the AsyncTimelockService, we shouldn't have to maintain backcompat; this means that these methods should ideally be replacing some current implementations (other than IdentifiedTime, which is new).

Put differently, refreshLockLeases should be defined in terms of leasableRefreshLockLeases, and probably only at the AsyncTimelockResource level (not here).

Same for the startTransaction one; there should simply not be a non-leaseable startIdentifiedAtlasDbTransaction at this part of timelock; it should be stripped out on the way to the client at the level above.

The method here should just be 'refreshLockLeases'.